js/common/utils/sort.js

```diff
@@ -0,0 +1,58 @@
+// Sort.js
+document.getElementById("sortLatest").addEventListener("click", function () {
+  sortPosts("latest");
+});
+
+document.getElementById("sortOldest").addEventListener("click", function () {
+  sortPosts("oldest");
+});
+
+document.getElementById("sortAZ").addEventListener("click", function () {
+  sortPosts("az");
+});
+
+document.getElementById("sortZA").addEventListener("click", function () {
+  sortPosts("za");
+});
+
+function sortPosts(sortBy) {
+  const postsContainer = document.getElementById("postsContainer");
+  const posts = Array.from(postsContainer.children);
+  // case names appended to id's in feed html
+  switch (sortBy) {
+    case "latest":
+      posts.sort((a, b) => {
+        const dateA = new Date(a.getAttribute("data-created"));
```

Comment:
I can't see the `date-created` attribute in the HTML?

Reply:
> I can't see the `date-created` attribute in the HTML?

No I sadly havent figured out how to do this. I'm struggling to implement this properly as I don't truly understand it.